src/main/java/com/coveo/spillway/SpillwayFactory.java

```diff
@@ -56,6 +56,20 @@ public SpillwayFactory(LimitUsageStorage storage, Clock clock) {
     this.clock = clock;
   }
 
+  /**
```

Comment:
Doesn't the `public final <T> Spillway<T> enforce(String resource, Limit<T>... limits)` method already cover this?

Reply:
It throws

Reply:
:sojava: